### ISSUE TITLE
NPE thrown if no tokenstores are configured

### ISSUE THREAD

**Issue by cwiechmann** [Axway-API-Management-Plus/apimanager-swagger-promote#32]
```Running API-Manager Promote version 1.4.1 ...
0      INFO                   App| ------------------------------------------------------------------------
1      INFO                   App| API-Manager Promote Version: 1.4.1
1      INFO                   App|
1      INFO                   App| To report issues or get help, please visit:
1      INFO                   App| https://github.com/Axway-API-Management-Plus/apimanager-swagger-promote
1      INFO                   App| ------------------------------------------------------------------------
886    ERROR                  App| Cant parse JSON-Config file(s)
com.axway.apim.lib.AppException: Cant parse JSON-Config file(s)
        at com.axway.apim.swagger.APIImportConfig.getImportAPIDefinition(APIImportConfig.java:126)
        at com.axway.apim.App.run(App.java:135)
        at com.axway.apim.App.main(App.java:38)
Caused by: com.axway.apim.lib.AppException: Can't find tokenstores: ....
        at com.axway.apim.swagger.api.properties.securityprofiles.SecurityDevice.initCustomPolicies(SecurityDevice.java:76)
        at com.axway.apim.swagger.api.properties.securityprofiles.SecurityDevice.<init>(SecurityDevice.java:42)
        at com.axway.apim.swagger.APIImportConfig.addDefaultPassthroughSecurityProfile(APIImportConfig.java:497)
        at com.axway.apim.swagger.APIImportConfig.getImportAPIDefinition(APIImportConfig.java:110)
        ... 2 more
Caused by: java.lang.NullPointerException
        at com.axway.apim.swagger.api.properties.securityprofiles.SecurityDevice.initCustomPolicies(SecurityDevice.java:70)
        ... 5 more
```

**Comment by cwiechmann** [comment]
The issue happens, if a user with Non-Admin-Role is used. With that, the REST-API-Call to load the Token-Stores (and other customer policies) fails.

**Comment by cwiechmann** [comment]
Changed the error handling to log a clearer error-message. When solving issue: #11 this will be improved with a generic approach.
CC: @rathnapandi

**Comment by cwiechmann** [comment]
Fixed with version 1.4.2
